Lib/test/test_free_threading/test_itertools_batched.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+import sys
+from threading import Thread, Barrier
+from itertools import batched
+from test.support import threading_helper
+
+
+class EnumerateThreading(unittest.TestCase):
+
+    @threading_helper.reap_threads
+    @threading_helper.requires_working_threading()
```

Comment:
use module level skip such as `threading_helper.requires_working_threading(module=True)`, that would avoid duplication for future tests